File: la_rive_admin/Models/Order.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using la_rive_admin.Models.Data.Enums.Order;

namespace la_rive_admin.Models
{
    [Table("Order")]
    public class Order
    {
        public string Id { get; set; }

        public string Store { get; set; }
        public string OrderNr { get; set; }
        public string OrderContent { get; set; }
        public double Amount { get; set; }
        public double RateAmount { get; set; }
        public int CountRate { get; set; }
        public DateTime Date { get; set; }
        public DateTime FirstPaymentDate { get; set; }

        public MeansOfPayment MeansOfPayment { get; set; }
        public PaymentMethode PaymentMethode { get; set; }
        public PaymentRate PaymentRate { get; set; }

        public DeliveryState DeliveryState { get; set; }
    }
}

File: la_rive_admin/ClientApp/src/app/models/order.ts
import { Item } from "../item/item";

export class Order {

  public Order() {
  }

  id: string;
  orderNr: string;
  store: string;
  orderContent: string;
  amount: number
  rateAmount: number;
  countRate: number;
  meansOfPayment: MeansOfPayment;
  paymentMethode: PaymentMethode;
  date: Date
  deliveryState: number;
  firstPaymentDate: Date;
}

export enum MeansOfPayment {
  Default,
  Paypal,
  Klarna,
  AfterPay,
  Visa,
  MasterCard,
  Transfer,
  Cash,
}

export enum PaymentMethode {
  Default,
  Total,
  Rate,
}
